testing/xerces-c/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Contributor: Andrew Bell <andrew.bell.ia@gmail.com>
+# Maintainer: Andrew Bell <andrew.bell.ia@gmail.com>
+pkgname=xerces-c
+pkgver=3.2.2
+pkgrel=0
+pkgdesc="Xerces-C++ is a validating XML parser written in a portable subset of C++."
+url="https://xerces.apache.org/index.html"
+arch="x86 x86_64"
+license="Apache-2.0"
+makedepends="
+    cmake
+    linux-headers
+    "
+install=""
+subpackages="$pkgname-doc"
+_pkgver=${pkgver//./_}
+_srcname="Xerces-C_${_pkgver}"
+source="https://github.com/apache/xerces-c/archive/${_srcname}.tar.gz"
+builddir="$srcdir/$pkgname-$_srcname"
+
+build() {
```

Comment:
If you add a prepare function like this:

```sh
prepare() {
    mkdir -p "$builddir"
    
    default_prepare
}
```
Then you can set `builddir` to `srcdir/$pkgname-$_srcname/build` and can remove the `mkdir build` and `cd` statements from the rest of the functions.

Reply:
That approach is going to cause patching to fail.

Reply:
Fine for now.